jmbitcoin/test/test_bip21.py

```diff
@@ -0,0 +1,74 @@
+import jmbitcoin as btc
+import pytest
+
+
+def test_bip21():
+
+    # These should raise exception because of not being valid BIP21 URI's
+    try:
```

Comment:
these checks can be done this way in pytest:

```python3
with pytest.raises(WhateverExceptionIsRaisedByTheCall):
    btc.decode_bip_21_uri('')
```
It's a bit more elegant than what you have here, although it's fine of course.

Reply:
Thanks, this is more elegant! I'm still learning Python. :)